aodncore/pipeline/handlerbase.py

```diff
@@ -44,8 +44,8 @@ class HandlerBase(object):
     :type input_file: str
 
     :param allowed_archive_path_regexes: List of allowed regular expressions of which
-        :py:attr:`PipelineFile.archive_path` must match at least one. If any non-matching values are found, the handler
-        will exit with an error during the publish step *before* publishing anything.
+        .. note:: :py:attr:`PipelineFile.archive_path` must match at least one. If any non-matching values are found,
```

Comment:
This has nothing to do with the csv harvester, and I don't think it's necessary. The ".. note::" text is inserted in the middle of a sentence, which is a bit odd.